ivy/functional/frontends/tensorflow/math.py

```diff
@@ -42,3 +42,10 @@ def divide(x, y, name=None):
 
 
 divide.unsupported_dtypes = {"torch": ("float16", "bfloat16")}
+
+
+def log_sigmoid(x, name=None):
+    return -ivy.softplus(-x)
+    # return ivy.log(ivy.sigmoid(x))
```

Comment:
Why is this a comment?

Reply:
I was trying the commented-out way first but saw in the documentation that it is implemented using softplus which is more robust. Forgot to delete the commented out code, will do that right now